### PR TITLE
update myself

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@sbillinge

File: db/people.yml [hunk 1]
```diff
@@ -293,7 +297,7 @@ cwright:
     - level: intermediate
       category: Experiments
       name: Electron Microscopy
-  title: Mr.
+  title: Dr.
```
Comment (sbillinge) [inline db/people.yml]: hah, I like these changes!

**Comment by sbillinge** [comment]
thanks @CJ-Wright

**Comment by CJ-Wright** [comment]
Should I have put my current association as a post doc?

**Pull request merged**